src/Components/productImage/ProductImage.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
-import image from '../sample.jpg';
 import './ProductImage.scss';
 
-const labelON = true ? 'label--on' : 'label--off'; 
+const ProductImage = ({image, label}) => {
+    const labelOn = (label = true) ? 'label--on' : 'label--off';
 
-const ProductImage = () => {
     return (
         <div className='image-box'>
-            <img className='product-image' src={image} alt='Product'/>
-            <div className={labelON}>Wyróżnione</div>
+            <img className='product-image' src={`${image}.jpg`} alt='Product'/>
+            <div className={labelOn}>Wyróżnione</div>
```

Comment:
FYI możesz tutaj wprost przekazać `label ? 'label--on' : 'label--off'` bez consta u góry. Przy bardziej rozbudowanych rzeczach warto wrzucać nie do kodu html, a przed returnem. 

Jakbyś się zdecydowała na to to można całość uprościć i wrzucić natychmiastowego returna: 
```
const ProductImage = ({image, label}) => (
    <div className='image-box'>
        <img className='product-image' src={`${image}.jpg`} alt='Product'/>
        <div className={label ? 'label--on' : 'label--off'}>Wyróżnione</div>
    </div>
)